src/components/media_manager/src/socket_streamer_adapter.cc

```diff
@@ -70,6 +70,8 @@ SocketStreamerAdapter::SocketStreamer::SocketStreamer(
     ip_(ip),
     port_(port),
     header_(header),
+    server_socket_(),
```

Comment:
Please see
- Scott Meyers, Effective c++ 3rd, Item 4: Make sure that objects are 
initialized before they’re used.
- McConnel, Code Complete, 2nd. In the Russian  edition, page 236 
(common rules for var init)